tests/core/vm/test_london.py

```diff
@@ -166,7 +166,7 @@ def test_revert_on_reserved_0xEF_byte_for_CREATE_and_CREATE2_post_london(
     )
 
     assert successful_create_computation.is_success
-    # assert only the appropriate gas is consumed, not all the gas. This falls within a range
+    # assert gas is consumed. This falls within a range
```

Comment:
I kind of liked the clarification you had in there before, like:

```suggestion
    # assert gas is consumed, but not all gas. This falls within a range:
```